test-cases/longevity/longevity-10gb-3h.yaml

```diff
@@ -21,3 +21,5 @@ space_node_threshold: 64424
 
 gce_n_local_ssd_disk_db: 2
 run_db_node_benchmarks: true
+
+append_scylla_args: '--commitlog-use-hard-size-limit=true'
```

Comment:
is this feature gonna be default in scylla ? do we want it default in SCT ?

Reply:
yes, it will be default in the future https://github.com/scylladb/scylladb/pull/10974.

But I changed here just for testing. will remove when PR will be ready